isilon_create_users.sh

```diff
@@ -21,189 +21,189 @@ CLUSTER_NAME=""
+      *)
+             echo "ERROR -- unknown arg $1"
+             usage
+             ;;
+    esac
+    shift;
 done
 
 if [ "$VERBOSE" == "y" ] ; then
-   set -x
+   set -x
 fi
 
 case "$DIST" in
-    "cdh")
-        SUPER_USERS="hdfs mapred yarn HTTP hbase"
-        SUPER_GROUPS="hadoop supergroup"
-        REQUIRED_USERS="$SUPER_USERS cloudera-scm accumulo flume hbase hive httpfs hue apache impala kafka kms keytrustee kudu llama oozie solr spark sentry sqoop sqoop2 zookeeper anonymous cmjobuser"
-        REQUIRED_GROUPS="$REQUIRED_USERS $SUPER_GROUPS"
-        PROXY_SUPER="impala flume hive hue oozie mapred"
-        PROXY_USERONLY="HTTP"
-        SMOKE_USER="cloudera-scm"
-        ;;
-    "hwx")
-        SUPER_USERS="hdfs mapred yarn hbase storm falcon tracer tez hive hcat oozie zookeeper ambari-qa flume hue accumulo hadoopqa sqoop spark mahout ranger kms atlas ams kafka zeppelin livy logsearch infra-solr activity_analyzer activity_explorer HTTP knox ambari-server"
-        SUPER_GROUPS="hadoop"
-        REQUIRED_USERS="$SUPER_USERS anonymous" 
-	if [ "$ZONE" != "System" ]; then
-          REQUIRED_USERS="$REQUIRED_USERS admin"
-        fi
-        REQUIRED_GROUPS="$REQUIRED_USERS $SUPER_GROUPS"
-        PROXY_SUPER="yarn livy hcat hbase flume hive oozie root"
-        PROXY_USERONLY="HTTP ambari-server knox"
-        SMOKE_USER="ambari-qa"
-        ;;
-    "bi")
-        SUPER_USERS="hdfs mapred hbase knox uiuser dsmadmin bigsheets ambari-qa rrdcached hive yarn hcat bigsql tauser bigr flume nagios solr spark sqoop zookeeper oozie bighome ams livy logsearch infra-solr activity_analyzer activity_explorer HTTP knox ambari-server" 
-	SUPER_GROUPS="hadoop"
-        REQUIRED_USERS="$SUPER_USERS anonymous"
-	if [ "$ZONE" != "System" ]; then
-          REQUIRED_USERS="$REQUIRED_USERS admin"
-        fi
-        REQUIRED_GROUPS="$REQUIRED_USERS $SUPER_GROUPS"
-        PROXY_SUPER="yarn livy hcat hbase flume hive oozie root"
-        PROXY_USERONLY="HTTP ambari-server knox"
-        SMOKE_USER="ambari-qa"
-        ;;
-    *)
-    echo "ERROR -- Invalid Hadoop distribution"
-        usage
-        ;;
+    "cdh")
+        SUPER_USERS="hdfs mapred yarn HTTP hbase"
+        SUPER_GROUPS="hadoop supergroup"
+        REQUIRED_USERS="$SUPER_USERS hive impala hue cloudera-scm accumulo flume httpfs apache kafka kms keytrustee kudu llama oozie solr spark sentry sqoop sqoop2 zookeeper anonymous cmjobuser"
+        REQUIRED_GROUPS="$REQUIRED_USERS $SUPER_GROUPS"
+        PROXY_SUPER="impala flume hive hue oozie mapred"
+        PROXY_USERONLY="HTTP"
+        SMOKE_USER="cloudera-scm"
+        ;;
+    "hwx")
+        SUPER_USERS="hdfs mapred yarn hbase storm falcon tracer tez hive hcat oozie zookeeper ambari-qa flume hue accumulo hadoopqa sqoop spark mahout ranger kms atlas ams kafka zeppelin livy logsearch infra-solr activity_analyzer activity_explorer HTTP knox ambari-server"
+        SUPER_GROUPS="hadoop"
+        REQUIRED_USERS="$SUPER_USERS anonymous root"
```

Comment:
I suspect we only want to add `root` when `--append-cluster-name` is used? This is odd, though, because such a user would not have super-user permissions...